holoviews/plotting/bokeh/__init__.py

```diff
@@ -191,6 +191,8 @@ def colormap_generator(palette):
 
 # Rasters
 options.Image = Options('style', cmap=dflt_cmap)
+options.Image = Options('plot', clipping_colors={'min': 'transparent',
+                                                 'max': 'transparent'})
```

Comment:
Why is 'max' transparent?  I don't think we want to avoid seeing the maximum pixels; seems like those should be in another color sometimes (e.g. red for a colormap not including red), but it's hard to think of a use for them being transparent.

Reply:
I think we discussed this already: we want anything out of the colormap bounds to show up with symmetry at both ends i.e mix and max. I am happy to change it again if there is a good reason for max to be treated different from min...